The short map from a convergent Cauchy approximation in a saturated metric space to its limit (#1402)
UniMath/agda-unimath: src/metric-spaces/dependent-products-metric-spaces.lagda.md

      ( P a)
      ( λ f → f a)
  is-short-ev-Π-Metric-Space ε x y H = H a

  short-ev-Π-Metric-Space :
    short-function-Metric-Space
      ( Π-Metric-Space A P)
      ( P a)
  short-ev-Π-Metric-Space = (λ f → f a) , (is-short-ev-Π-Metric-Space)
```

> ### The product of saturated metric spaces is saturated

Reviewer: ```suggestion
### Dependent products of saturated metric spaces are saturated
```
